### PR TITLE
Removed default project from issues templates

### LINKED ISSUE #11: [Feat]: Update issues template to provide more information
### Feature Type

- [X] Adding new functionality

- [ ] Changing existing functionality

- [ ] Removing existing functionality

### Problem Description

Issues forms don't give enough context when asking for information and make too much information mandatory.

### Feature Description

- Provide some context and explanation at the top of each issue template using the [markdown element](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#markdown), should explain what the issue is intended for, thank the user for filling it out and provide some context to why each of the fields is required
- Make some of the fields optional e.g. Feature Template - Alternative Solutions section
- Remove default project from the issues templates

### Alternative Solutions

NA

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by MattBuckley-TfN**
# Changes

We don't want all issues across all repositories to be sent to the CAF project.

- Closes #11 

# Tasks

- [x] Have unittests been added (testing individual functions and their edge cases) - NA
- [x] Has documentation (including user guide) been updated - NA
- [x] Have new dependencies been added (or changed) in relevant `requirements.txt` - NA
- [x] Code linting, tests and other workflows pass - NA

### CODE REVIEW

**Comment by MattBuckley-TfN** [comment]
@BenTaylor-TfN - not ready for review yet need to address stuff in #11

**Comment by MattBuckley-TfN** [comment]
@BenTaylor-TfN - I've added some descriptions in markdown elements to the top of each issue template to help explain the reason for the questions and give some context.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -25,8 +34,9 @@ body:
     attributes:
       label: Reproducible Code Example
       description: >
-        Please follow [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports)
-        on how to provide a minimal, copy-pastable example.
+        Please provide a simple example of code which can reproduce the bug.
+        [This guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports)
+        explains how to provide a minimal, copy-pastable example.
```
Comment (BenTaylor-TfN) [inline .github/ISSUE_TEMPLATE/bug.yml:39]: I really like this.

I feel there's something we could add in the handbook that could provide people with more context about why we need the minimal bug report, and how we use it. i.e., Make tests to ensure the problem never comes back.

That would, of course, link back to this guide above

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -2,13 +2,22 @@ name: Bug Report
...
 
 body:
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for taking the time to fill this bug report. 
+
+        Please provide as much detail explaining the problem and expected result.
+        The questions below show the information we need to be able to properly
+        reproduce and fix the bug.
```
Comment (BenTaylor-TfN) [inline .github/ISSUE_TEMPLATE/bug.yml]:
```suggestion
        Thank you for taking the time to fill out this bug report. 

        Help us help you! Please explain the problem and your expected result.
        The questions below show the information we need to properly
        reproduce and fix the bug, solving your problem quicker.
```

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 1 above)
Comment (MattBuckley-TfN) [reply .github/ISSUE_TEMPLATE/bug.yml:39]: That guide has come directly from the pandas issue template, do you think it would be within the contribution guidelines to give advice for what types of contributions people can give e.g. code changes vs bug reports?

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 3]
```diff
@@ -2,9 +2,16 @@ name: Documentation Improvement
...
-projects: ["Transport-for-the-North/48"]
 
 body:
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for taking time to suggest this documentation improvement.
+
+        Please provide information to explain what is unclear, or missing from,
+        the documentation and an explanation of your suggestion for improvement.
```
Comment (BenTaylor-TfN) [inline .github/ISSUE_TEMPLATE/documentation.yml]:
```suggestion
        Please explain what is unclear or missing from the documentation and,
        if possible, how to improve. 
```

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 4]
```diff
@@ -30,7 +37,10 @@ body:
     attributes:
       label: Documentation problem
       description: >
-        Please provide a description of what documentation you believe needs to be fixed/improved
+        Please provide a description of what documentation you believe needs to be
+        added, fixed or improved.
```
Comment (BenTaylor-TfN) [inline .github/ISSUE_TEMPLATE/documentation.yml]:
```suggestion
        Please describe which part of the documentation needs to be
        added, fixed or improved.
```

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 5]
```diff
@@ -2,9 +2,22 @@ name: Feature Request
 description: Suggest an new feature / change to the functionality.
 title: "[Feat]: "
 labels: [enhancement]
-projects: ["Transport-for-the-North/48"]
 
 body:
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for taking time to suggest this feature, or improvement.
```
Comment (BenTaylor-TfN) [inline .github/ISSUE_TEMPLATE/feature.yml]:
```suggestion
        Thank you for taking time to suggest this feature or improvement.
```

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 6]
```diff
@@ -2,9 +2,22 @@ name: Feature Request
...
+
+        Please provide information about the problem this feature is trying
+        to improve before explaining the details of the feature itself. If
+        there is a current workaround please include any information about this
+        and why the new feature would be better.
+
+        For any additional information about the feature such as inputs, outputs
+        or an example implementation, please provide that for the
+        "Additional Context" question.
```
Comment (BenTaylor-TfN) [inline .github/ISSUE_TEMPLATE/feature.yml]:
```suggestion
        Please describe the problem this feature will improve before explaining the 
        details of the feature itself. If there is a current workaround please include 
        any information about this and why the new feature would be better.

        Please use the "Additional Context" question to note if this feature
        requires specific inputs, outputs, or you have an example implementation.
```

**Review by BenTaylor-TfN — APPROVED** [review]
